pkgs/applications/networking/qusb2snes/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, mkDerivation, git, qmake, qtbase, qtserialport, qtwebsockets }:
+mkDerivation rec {
+  pname = "QUsb2snes";
+  version = "0.7.19.1";
+
+  src = fetchTarball {
+    url = "https://github.com/Skarsnik/QUsb2snes/archive/v${version}.tar.gz";
+    sha256 = "1fj09sj072j3055dfjfbhzil0hqz2bv3mwv09477iz17bvifhf3k";
+  };
```

You should be using `fetchFromGitHub`.
```suggestion
  src = fetchFromGitHub {
    owner = "Skarsnik";
    repo = "QUsb2snes";
    rev = "v${version}";
    sha256 = "0000000000000000000000000000000000000000000000000000000000000000";
  };
```

---

Ha, didn't expect the hash to be the same, but these two calls seem equivalent in result.